Assets/MRTK/Core/Definitions/SpatialAwareness/MixedRealitySpatialAwarenessMeshObserverProfile.cs

```diff
@@ -89,7 +89,52 @@ public class MixedRealitySpatialAwarenessMeshObserverProfile : BaseSpatialAwaren
 
         public PhysicMaterial PhysicsMaterial => physicsMaterial;
 
-        
+        [SerializeField]
+        [Tooltip("Optional prefab that is added to the runtime spatial mesh hierarchy.  This prefab will only" +
+            " be instantiated and appended to the hierarchy in a build, the behavior associated with this property is not" + 
+            " enabled in editor.\n" +
+            "\n" +
+            "Default Runtime Spatial Awareness Hierarchy: \n" +
+            "\n" +
+            "Spatial Awareness System \n" +
+            "   Windows Mixed Reality Spatial Mesh Observer\n" +
+            "       Spatial Mesh - ID\n" +
+            "       Spatial Mesh - ID\n" +
+            "       ...\n" +
+            "\n" +
+            "Runtime Spatial Awareness Hierarchy with this prefab:\n" +
+            "\n" +
+            "Spatial Awareness System \n" +
+            "   Runtime Spatial Mesh Prefab\n" +
+            "       Windows Mixed Reality Spatial Mesh Observer\n" +
+            "           Spatial Mesh - ID\n" +
+            "           Spatial Mesh - ID\n" +
+            "            ...\n")]
+        private GameObject runtimeSpatialMeshPrefab = null;
+
+        /// <summary>
+        /// Optional prefab that is added to the runtime spatial mesh hierarchy.  This prefab will only
+        /// be instantiated and appended to the hierarchy in a build, the behavior associated with this property is not 
+        /// enabled in editor.
+        /// 
+        /// Runtime Spatial Awareness Hierarchy without this prefab:
+        /// 
+        /// Spatial Awareness System 
+        ///     Windows Mixed Reality Spatial Mesh Observer
+        ///         Spatial Mesh - ID
+        ///         Spatial Mesh - ID
+        ///         ...
+        ///         
+        /// Runtime Spatial Awareness Hierarchy with this prefab:
+        /// 
+        /// Spatial Awareness System 
+        ///         Runtime Spatial Mesh Prefab
+        ///             Windows Mixed Reality Spatial Mesh Observer
+        ///                 Spatial Mesh - ID
+        ///                 Spatial Mesh - ID
+        ///                 ...
+        /// </summary>
+        public GameObject RuntimeSpatialMeshPrefab => runtimeSpatialMeshPrefab;
```

Comment:
Do we want to rename this something like...`RuntimeSpatialMeshParentPrefab`? I could see some confusion in thinking this is spawned for each detected mesh. _Though_, you do a great job explaining it, its purpose, and where it's used in the summary tag so it's probably fine.

Reply:
It might be confusing to have the name `RuntimeSpatialMeshParentPrefab` when we also have this property `SpatialAwarenessObjectParent`:

```
CoreServices.SpatialAwarenessSystem.SpatialAwarenessObjectParent
```
the `SpatialAwarenessObjectParent` is at the top of the hierarchy

```
 Spatial Awareness System  (SpatialAwarenessObjectParent)
     Runtime Spatial Mesh Prefab (RuntimeSpatialMeshPrefab)
         Windows Mixed Reality Spatial Mesh Observer
             Spatial Mesh - ID
             Spatial Mesh - ID
             ...
```

@keveleigh  what about renaming it to `RuntimeSpatialMeshObserverParentPrefab` and we end up with this hierarchy:

```
 Spatial Awareness System  (SpatialAwarenessObjectParent)
     Runtime Spatial Mesh Prefab (RuntimeSpatialMeshObserverParentPrefab)
         Windows Mixed Reality Spatial Mesh Observer
             Spatial Mesh - ID
             Spatial Mesh - ID
             ...
```

Adding "ObserverParent" would be more specific to the hierarchy and may limit confusion between the `SpatialAwarenessObjectParent` vs a `RuntimeSpatialMeshParentPrefab`.

Reply:
Yeah, I think I'm happy either way! Adding both `Observer` and `Parent` might get a little long...and the docs definitely explain what this is, even if we just leave the name as-is.